Documentation/component-description.md

```diff
@@ -36,30 +38,34 @@ In this page, we focus on the status of each elements. For requirements and desi
     - TODO: Recurrent models. The first target unit-test is LSTM.
   - Tensorflow-Lite (stable)
   - Custom (stable)
-  - Tensorflow (The next incoming TODO)
+  - Tensorflow (WIP. Not working properly as of 2018/10/13)
   - Other NNFW TBD (caffe2, caffe, ...)
-- [tensor\_sink](gst/tensor_sink/) (stable)
-  - TODO: ROS integration (not determined if this will be yet another element or not)
-- [tensor\_transformer](gst/tensor_transformer/) (experimental)
-  - Only fraction of supposed features are supported.
-  - TODO: a lot more features to be implemented. Even the implemented feature is incomplete.
-  - Note: a few chosen partial features will be added & tested for Project:NS.
-- [tensor\_merge](gst/tensor_merge/) (planned)
-- [tensor\_split](gst/tensor_split/) (experimental)
-  - Basic feature is implemented. Need test & verficiation.
-  - Note: a few chosen partial features is being implemented & tested for Project:NS.
-- [tensor\_decoder](gst/tensor_decoder/) (experimental)
-  - An example feature is implemented as a template.
-  - TODO: prepare tensor-to-media transforms for many popular neural network models.
-    - Adding more functionalities for sample applications: jihyuck-park
-- [tensor\_mux](gst/tensor_mux/) (stable)
-- [tensor\_demux](gst/tensor_demux/) (stable)
-- [tensor\_source](gst/tensor_source/) (planned)
-- [tensor\_save](gst/tensor_save/) (planned)
-- [tensor\_load](gst/tensor_load/) (planned)
-- [tensor\_aggregator](gst/tensor_aggregator) (experimental)
-  - Basic features being implemented without enough test cases.
-  - Note: a few choen partial features is being implemented & tested for Project:NS.
+- [tensor\_sink](../gst/tensor_sink/) (stable)
+- [tensor\_transformer](../gst/tensor_transformer/) (stable, but with a lot of WIP items)
```

Comment:
rename to 'tensor_transform'
[tensor\_transform](../gst/tensor_transform/)

Reply:
Fixed